include/framework/everest.hpp

```diff
@@ -74,6 +79,7 @@ class Everest {
     /// \brief Allows a module to indicate that it provides the given command \p cmd
     ///
     void provide_cmd(const std::string& impl_id, const std::string& cmd_name, const JsonCommand& handler);
+    void provide_cmd(const std::string& impl_id, const std::string& cmd_name, const JsonCommandMultiple& handler);
```

Comment:
should we just have this as the generic signature and just change the generated `handle_{cmd}` signature based on the quantity parameter?